src/assets/less/components/jumbotron.less

```
@@ -51,31 +55,63 @@
+				display: none;
 			}
+		}
 
-			&_circle-one {
-				height: 500px;
-				top: -14%;
-				left: -30%;
+		.jumbotron_globe {
+			display: none;
+			--globe-size: min(900px, calc(100vw - 64px));
+			left: 50%;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)